app/views/social_networking/homes/show.html.erb

```diff
@@ -59,65 +59,90 @@
         class="list-group-item"
         id="{{ item.className }}-{{ item.id }}">
       <p>
-        <div ng-if="!item.data.summary.length">
-          <img width="40px;" ng-src="{{ home.profileIconSrcFor(item) }}"> {{ item.summary }}
+        <div class="media feed-item-displayname" ng-if="!item.data.summary.length">
```

Comment:
For future reference we use `feed-item-displayname` in "home_tool_spec.rb" and "profile_spec.rb" of SunnySide.